typed-racket-lib/typed-racket/static-contracts/combinators/control.rkt

```diff
@@ -52,16 +50,11 @@
 
 
 (define (pt-seq-map f seq)
-  (match seq
-   [(pt-seq vals call-cc)
-    (define (f* a) (f a 'invariant))
-    (pt-seq
-      (map f* vals)
-      (and call-cc (map f* call-cc)))]))
+  (match-define (pt-seq vals call-cc) seq)
+  (define (f* a)
```

Comment:
@sorawee why is this on two lines?

Reply:
I would be interested to find that discussion; I strongly disagree.

Reply:
It's @Metaxal. I'm very lucky that I wrote your name in the commit message 😂

https://github.com/sorawee/fmt/commit/831bb4cb6aac95c192be2a9d1b9bcd0f4519b84b

Reply:
Haha indeed! Though "Suggestion by" sounds a bit different from "gathered supporters to NOT do" ;)

Again, I don't feel particularly strongly about this, although I do have a preference for two lines so that the function header stands out. I'm not going to die on this hill though.

Reply:
It was also me. I was (and still am) of the strong opinion that function definitions should never be on a single line. Otherwise it's hard to tell them apart from variable definitions.